### PR TITLE
cleanupbuddy 1.0 (new cask)

### PR DESCRIPTION
**Pull request by HuaDeity**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/c/cleanupbuddy.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "cleanupbuddy" do
+  version "1.0"
+  sha256 :no_check
+
+  url "https://sparkle.cleanupbuddy.app/CleanupBuddy-latest.dmg"
```
Comment (bevanjkay) [inline Casks/c/cleanupbuddy.rb:5]:
```suggestion
  url "https://sparkle.cleanupbuddy.app/CleanupBuddy_v#{version.csv.first}-#{version.csv.second}.dmg"
```

Use versioned `url`

File: Casks/c/cleanupbuddy.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  name "CleanupBuddy"
+  desc "Clean keyboard and trackpad"
+  homepage "https://cleanupbuddy.app/"
+
+  livecheck do
+    url "https://sparkle.cleanupbuddy.app/appcast.xml"
+    strategy :sparkle, &:short_version
```
Comment (bevanjkay) [inline Casks/c/cleanupbuddy.rb:12]:
```suggestion
    strategy :sparkle
```

File: Casks/c/cleanupbuddy.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+cask "cleanupbuddy" do
+  version "1.0"
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/c/cleanupbuddy.rb:3]: Add `sha256` value based on adding a versioned url

**Review by bevanjkay — DISMISSED** [review]
Please see review comments

**Review by miccal — APPROVED** [review]

**Pull request merged**